Pulse.Core/Connections/UdpChannel.cs

```diff
@@ -26,6 +26,8 @@ private async Task ListenAsync()
 
             if (message.Buffer.All(b => b == 0) && message.Buffer.Length is 472)
             {
+                // TODO: Remove part from here...
+                // TODO: Assert that a packet length is exactly XXX (number we will decide later), this is important.
```

Comment:
Isn't that what we did already?

Reply:
It should not be here, packets are encrypted..